templates/repo/issue/new_form.tmpl

```diff
@@ -236,5 +236,6 @@
 					{{end}}
 				</div>
 		</div>
+		<input type="hidden" name="redirect_on_project" value="{{.redirect_on_project}}">
```

Comment:
Maybe we can call this variable `redirection_after_creation` (now we only have 2 choices, empty or `project`), then developers in future will have a chance to introduce more redirection pages.

Reply:
Yes good idea, it's done :+1:

Reply:
> Maybe we can call this variable `redirection_after_creation` (now we only have 2 choices, empty or `project`), then developers in future will have a chance to introduce more redirection pages.

I think it can be even shorter, what do you think about `redirect`~~`ion`~~`_after_creation`?